arches/app/etl_modules/import_single_csv.py

```diff
@@ -97,14 +98,19 @@ def write(self, request):
 
         fieldnames = request.POST.get("fieldnames").split(",")
         column_names = [fieldname for fieldname in fieldnames if fieldname != ""]
+        id_label = "Use as an id"
```

Comment:
I don't think this is true anymore right? We are just using the id_label down on line 164. Maybe the id_label assignment on 164 should be moved up here?